frontend/src/components/samples/SamplesDetailContent.js

```diff
@@ -73,6 +77,15 @@ const SamplesDetailContent = ({samplesByID, sampleKindsByID, containersByID, ind
   if (isLoaded && !sample.versions && !sample.isFetching)
     listVersions(sample.id);
 
+  if (isLoaded && !isProcessesEmpty && !sample.isFetching) {
+    sample.processes_samples.forEach((id, i) => {
+      withProcess(processesByID, id, process => process.id);
+    })
+    sample.processes_samples.forEach((id, i) => {
+      sampleProcesses.push(processesByID[id])
+    })
```

Comment:
`const sampleProcesses = sample.processes_samples.map(id => processesByID[id])`?

Btw why is that thing called `processes_samples`? I don't understand the double plural.

Reply:
Its the ProcessSample objects associated to a Sample. The model name is ProcessSample.. I guess I could just rename it 'ProcessSamples' if you prefer

Reply:
But are those things processes or samples? If those are processes I'd just call them processes.

Reply:
There are 3 models: Process, Sample and ProcessSample. Here we want to represent ProcessSample objects. ( I know this is confusing, even now with the Process page listing rows of ProcessSample, thats definitely an issue imo ...)